codalab/rest/workers.py

```diff
@@ -28,7 +28,8 @@ def checkin(worker_id):
     socket_id = local.worker_model.worker_checkin(
         request.user.user_id,
         worker_id,
-        request.json.get("tag"),
+        # TODO(nfliu): deprecate passing "tag" in the request. The right key is "tags".
```

Comment:
Can we make a GitHub issue for this, then link to the issue in the comment, too?